highway.go

```
@@ -56,8 +57,10 @@ func main() {
 	chainData := new(chaindata.ChainData)
 	chainData.Init(common.NumberOfShard)
 
+	whitelisthw, hostPriKey, err := key.GenWhiteList([]byte(conf.PrivateSeed), conf.HighwayIndex, common.NumberOfHighway)
```

it's difficult to input byte 128-255 because privateseed is string (from config)
should we receive privateseed as base64 encoded and decode it?

---

also, err is not handled